PeripheralController.swift

```diff
@@ -1,119 +1,112 @@
 import CoreBluetooth
 
-public struct PeripheralCharacteristicsDataV2: Codable {
-    var mp: String // phone model of peripheral
-    var id: String // tempID
-    var o: String // organisation
-    var v: Int // protocol version
+public struct PeripheralCharacteristicsData: Codable {
+    var modelP: String // phone model of peripheral
+    var msg: String // tempID
+    var org: String
+    var v: Int
 }
 
-public class PeripheralController: NSObject {
+let TRACER_SVC_ID: CBUUID = CBUUID(string: "\(PlistHelper.getvalueFromInfoPlist(withKey: "TRACER_SVC_ID") ?? "B82AB3FC-1595-4F6A-80F0-FE094CC218F9")")
+let TRACER_SVC_CHARACTERISTIC_ID = CBUUID(string: "\(PlistHelper.getvalueFromInfoPlist(withKey: "TRACER_SVC_ID") ?? "B82AB3FC-1595-4F6A-80F0-FE094CC218F9")")
+let ORG_ID = "<your org id here>"
```

Comment:
This does not appear to be used anywhere.

Reply:
Android sends it in the payload: https://github.com/AU-COVIDSafe/mobile-android/blob/b827cf3ccef72a3d38c6fc37466a99868823540f/app/src/main/java/au/gov/health/covidsafe/streetpass/StreetPassWorker.kt#L595

value is `AU_DTA`

Reply:
That’s `BluetoothConfig.OrgID` here: https://github.com/bok-/covidsafe-compare/blob/f8580c4faea732ad15bc3f1375a9b1f1083195f4/BluetraceConfig.swift#L15